test/constants/transactions.js

```diff
@@ -25,4 +25,62 @@ const transaction = index => ({
 
 const transactions = Array(30).fill(1).map((item, index) => transaction(index));
 
+const sampleTransaction = {
+  moduleID: 5,
```

Comment:
These are not constants, we should create fixtures and use them as fixtures.

Reply:
Fixed